### PR TITLE
Update phrasing in README

### PR DESCRIPTION
**Pull request by mathiasbynens**
We’re at a point where our initial “v1” spec roadmap has largely been completed, BiDi has several browser implementations, and popular ecosystem solutions are increasingly relying on WebDriver BiDi to unlock new use cases (like logging). It feels like a good time to update some of the phrasing in the README to reflect this state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ Now you can run in your terminal:
 ./scripts/test.sh
 ```
 
-This script will install required npm and cargo packages (if not installed yet)
-and generate the CDDL files for the remote end (`remote.cddl`) and the client
+This script installs required npm and cargo packages (if not installed yet)
```
Comment (thiagowfx) [inline README.md:45]: optional drive-by: `npm` and `cargo`

**Review by thiagowfx — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mathiasbynens) [reply README.md:45]:
```suggestion
This script installs required `npm` and `cargo` packages (if not installed yet)
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # WebDriver BiDi
 
-WebDriver BiDi is a proposed bidirectional protocol for browser automation,
+WebDriver BiDi is a bidirectional protocol for browser automation,
 building on and extending [WebDriver](https://w3c.github.io/webdriver/).
 
-WebDriver BiDi is not ready. Here's what we have so far:
+For more info, consult these resources:
```
Comment (whimboo) [inline README.md:6]: Should we say that it is a living spec and continuously gets new features added?

**Review by whimboo — APPROVED** [review]
Looks good to me. Thanks Mathias!
